pkg/controllers/management/usercontrollers/controller.go

```diff
@@ -210,6 +210,13 @@ func (c *ClusterLifecycleCleanup) createCleanupClusterRole(userContext *config.U
 			APIGroups: []string{"batch"},
 			Resources: []string{"jobs"},
 		},
+		// The job checks for the presence of the rancher service first
+		rbacV1.PolicyRule{
+			Verbs:         []string{"list", "get"},
```

Comment:
Is list required? The call is a `GET` that checks for the service

Reply:
Fixed